shared/base.js

```diff
@@ -53,12 +53,12 @@ function setupListeners(reporterID, options) {
   });
   chrome.tabs.onUpdated.addListener(enableOrDisable);
   chrome.browserAction.onClicked.addListener(tab =>
-    reportIssue(tab, reporterID, tabId)
+    reportIssue(tab, reporterID, tab.tabId)
```

Comment:
Almost!

If you read the documentation, https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/Tab (or the previous Chrome link (it's the same), the tab ID lives on the `id` property. So you need to pass in `tab.id` here.

Reply:
I'm so sorry, I completely misunderstood that! Shall make the change asap.

Reply:
Btw, the reason I had written that was because I saw this
>  chrome.browserAction.disable(tab.tabId);

and I thought thats how its done. Cany you please tell me whats the difference here? Is it that `tab.tabId` is the id of the existing one and ` tab.id` is the id of the new tab?